connect to the database

Mylaaaaa opened the pull request:
I added some comments for CSS and successfully connected our project to the database, you can check @YY-05 @Chenxi219

Mylaaaaa committed:
add some comments for CSS

Mylaaaaa committed:
connect the database

YY-05 commented:
Through testing, I found that we have done a very good job in this part and have successfully connected to the database.

Chenxi219 commented:
After testing, I found that it can run.

Mylaaaaa committed:
connect booking, login, register to the database

Mylaaaaa committed:
fixed the receipt bugs

When a booking is submitted successfully, all booking messages will show in the receipt
